WHO.js

```diff
@@ -0,0 +1,359 @@
+{
+	"translatorID": "cd587058-6125-4b33-a876-8c6aae48b5e8",
+	"label": "WHO",
+	"creator": "Mario Trojan, Philipp Zumstein",
+	"target": "^http://apps\\.who\\.int/iris/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-09-02 08:58:59"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Mario Trojan
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+// attr()/text() v2
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}
+function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+
+
+function detectWeb(doc, url) {
+	if (url.includes("/handle/") && text(doc, 'div.item-summary-view-metadata')) {
+		var type = attr(doc, 'meta[name="DC.type"]', 'content');
+		//Z.debug(type);
+		if (type && type.includes("articles")) {
+			return "journalArticle";
+		}
+		if (type && (type.includes("Book") || type.includes("Publications"))) {
+			return "book";
+		}
+		return "report";
+	} else if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+}
+
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+
+	var rows = doc.querySelectorAll('h4.artifact-title>a');
+	for (let i=0; i<rows.length; i++) {
+		let href = rows[i].href;
+		var title = rows[i].textContent;
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+
+function scrape(doc, url) {
+	// copy meta tags in body to head
+	var head = doc.getElementsByTagName('head');
+	var metasInBody = ZU.xpath(doc, '//body/meta');
+	for (let i=0; i<metasInBody.length; i++) {
```

Comment:
FWIW, I think Zotero house style now prefers `for (let item of array)` to cycle through arrays unless you need the index. I'll change that here, but I don't think it's worth it going through translators since there's no effective difference.